pkg/api/rest/client.go

```diff
@@ -0,0 +1,82 @@
+package rest
+
+import (
+	"net/url"
+	"strings"
+
+	"github.com/pingcap/ticdc/pkg/httputil"
+	"golang.org/x/time/rate"
+)
+
+type Interface interface {
```

Comment:
how about `RestClientInterface`

IMHO, When define a `interface`, we should care more about the `behavior`, not the `state`. so that, I do not recommend the method `GetRateLimiter` in the interface.

Reply:
> how about `RestClientInterface`

This interface will be changed later.

> IMHO, When define a `interface`, we should care more about the `behavior`, not the `state`. so that, I do not recommend the method `GetRateLimiter` in the interface.

The client-side rate limiter is not ready. I will remove the relevant code later.